packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -333,7 +333,11 @@ module.exports = function(log, config, oauthdb) {
     return {
       html: localized.html,
       language: translator.language,
-      subject: translator.gettext(message.subject),
+      subject: translator.format(
```

Comment:
Since there's now more than one subject that includes data, I moved the call to `format` up here to eliminate duplicate calling code.